src/cow-react/common/hooks/useParameterizedMainMenu.ts

```diff
@@ -0,0 +1,20 @@
+import { BasicMenuLink, isBasicMenuLink, MAIN_MENU, MenuTreeItem } from 'constants/mainMenu'
```

Comment:
Will we keel the `/common` or for we plan to create a `/modules/common`

Reply:
Already discussed, will change the project structure after merge